pkg/replication/replicator.go

```diff
@@ -143,10 +143,13 @@ func (r replicator) initializeReplicationWithCheck(
 		errCh:     errCh,
 		replicationCancelledCh: make(chan struct{}),
 		replicationDoneCh:      make(chan struct{}),
+		enactedCh:              make(chan struct{}),
 		quitCh:                 make(chan struct{}),
 		concurrentRealityRequests: make(chan struct{}, concurrentRealityRequests),
 		timeout:                   r.timeout,
 	}
+	// To make a closed channel
+	close(replication.enactedCh)
```

Comment:
furthermore, if the channel starts out as closed, `Enact()` will panic because you can't close an already closed channel: https://play.golang.org/p/spvZ45hOlM